assets/script/script.js

```diff
@@ -57,18 +59,13 @@ function apiCalls(city) {
 }
 
 function displayWeather(response) {
-  $(".nameTemp").html(response.name + " " + response.main.temp + " °F");
-  $(".weatherDay").html(`It's a ${weatherWord} day`);
-  let weatherIcon = response.weather[0].icon;
-  let iconUrl = "https://openweathermap.org/img/w/" + weatherIcon + ".png";
-  weatherWord = response.weather[0].main;
-  let divEl = $("<div>");
-  let imgEl = $("<img>");
-
-  imgEl.attr("src", iconUrl);
-  divEl.append(imgEl);
-  imgEl.attr("src", iconUrl);
-  $("#currentWeather").append(divEl);
+  console.log(response);
```

Comment:
@Meechlouch I might be wrong, but it appears lines 60-71 will be deleted.  If this is true, are these lines of code needed?

Reply:
Yes, I shortened the coded, so it is ok to erase it.

Reply:
@Meechlouch, thanks for the clarification and for tidying the code up.  I'll approve this PR.